beginners-yoga-course.html

```diff
@@ -0,0 +1,82 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Prema Healing Cleveland</title>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Signika:300">
+  <link rel="stylesheet" href="main.css">
+  </head>
+<body>
+<div class="pageWrap">
+
+<!-- HEADER -->
+<div class="gridCell--row gridCell--padded gridCell--row--justifyCenter">
+  <header class="pageHeader"><a href="index.html">
+    <h1 class="headline pageHeader__headline align--center">
+      <span class="headline__line pageHeader__primary inverted">Prema Healing</span>
+      <span class="headline__line pageHeader__secondary">Cleveland</span>
+    </h1></a>
+  </header>
+</div>
+
+<div class="gridCell--row program__row gridCell--row--justifyCenter">
+
+  <!-- COL A -->
+  <div class="gridCell--col gridCell--padded program__col">
+    <article class="program program-wide">
+
+      <h2 class="headline inverted uppercase program__title">Beginners Yoga Course</h2>
+
+      <div class="meta uppercase">
+        <p class="meta__item">
+          <span class="unbreakable">A five week course beginning April 13th – May 11th.</span>
+        </p>
+        <p class="meta__item">Wednesdays 6:30 – 7:30pm</p>
+      </div>
+
+      <div class="content">
+        <p>This course is a step-by-step introduction to Sivananda Classical Hatha Yoga. Students will experience the Five Points of Yoga including proper exercise, breathing, diet, relaxation, and meditation. Sivananda yoga guides students through asanas, while relaxing between each posture. Classes begin with chanting, pranayama breathing, sun salutations and go into the 12 basic asanas. This course walks students through at their own pace and completion of the course will provide a full understanding of the Sivanada Open Class.</p>
+        <p>Course is limited to 10 students per course.</p>
+        <p class="uppercase vertical-padding"><strong>Register by april 1st to save your spot!</strong></p>
+      </div>
+
+      <section class="additionalInfo">
+        <p class="inverted"><a href="https://docs.google.com/forms/d/1TUbJxIprhOcw02V6gS4UUgb7r8Kxm6rBtWOteg62ddw/viewform" target="_blank">Sign Up Here!</a></p>
```

Comment:
Small thing but the `inverted` class doesn't look necessary here since the `a` will already be Walter White.